go/flowctl/cmd-apply.go

```diff
@@ -182,46 +182,49 @@ func scopeToPathAndPtr(dir, scope string) (path, ptr string) {
 	return path, ptr
 }
 
-func applyCaptureShards(built *bindings.BuiltCatalog, client pc.ShardClient, shards int) error {
+func applyCaptureShards(built *bindings.BuiltCatalog, client pc.ShardClient, shards int, commonsRevision int64) error {
```

Comment:
nit: `taskRevision` (or `taskCreated` to align w/ label?) would be more accurate. conceivably the `taskRevision` and `commonsRevision` could be different.